trailblazer/store/filters/analyses_filters.py

```diff
@@ -48,16 +60,20 @@ def apply_analysis_filter(
     filter_functions: List[Callable],
     analysis_id: Optional[int] = None,
     case_id: Optional[str] = None,
+    comment: Optional[str] = None,
     started_at: Optional[datetime] = None,
     status: Optional[str] = None,
-    statuses: Optional[List[str]] = [],
+    statuses: Optional[List[str]] = None,
 ) -> Query:
     """Apply filtering functions and return filtered results."""
+    if statuses is None:
+        statuses = []
```

Comment:
Does this not conflict with the ´status´ argument, do we need to have both?

Reply:
We need both to do this:
```
def filter_analyses_by_status(analyses: Query, status: TrailblazerStatus, **kwargs) -> Query:
    """Filter analyses by status."""
    return analyses.filter(Analysis.status == status)


def filter_analyses_by_statuses(analyses: Query, statuses: List[str], **kwargs) -> Query:
    """Filter analyses by statuses."""
    return analyses.filter(Analysis.status.in_(statuses))

´´´
I think it is more readable with 2 functions and not coerce  str to a list